webhooks/pod_webhook.go

```diff
@@ -139,6 +159,30 @@ func (m *PodMutator) createFSConfigSpec(ctx context.Context, req admission.Reque
 	return featureFlagSourceSpec, 0, nil
 }
 
+func (m *PodMutator) createFSInProcessConfigSpec(ctx context.Context, req admission.Request, annotations map[string]string, pod *corev1.Pod) (*api.FeatureFlagInProcessConfigurationSpec, int32, error) {
+	// Check configuration
+	fscNames := []string{}
+	val, ok := annotations[fmt.Sprintf("%s/%s", common.OpenFeatureAnnotationPrefix, common.FeatureFlagInProcessConfigurationAnnotation)]
+	if ok {
+		fscNames = parseList(val)
+	}
+
+	featureFlagSourceSpec := api.FeatureFlagInProcessConfigurationSpec{}
+
+	for _, fscName := range fscNames {
+		ns, name := utils.ParseAnnotation(fscName, req.Namespace)
+
+		fc, err := m.getFeatureFlagInProcessConfiguration(ctx, ns, name)
+		if err != nil {
+			m.Log.V(1).Info(fmt.Sprintf("FeatureFlagInProcessConfiguration could not be found for %s", fscName))
```

Comment:
at this point the returned `err` can be either a `NotFound`, but also any other kind of error - should we distinguish between them here and also include the error in the log message? Also, let's include the namespace in the log to correctly relate the error to the request